frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -31,6 +31,17 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
     formatAssessed = '';
   }
 
+  let improvments = 0;
+  if (parcelInfo?.assessedBuilding) {
+    improvments = parcelInfo?.assessedBuilding;
+  } else if (parcelInfo?.buildings?.length >= 1) {
+    parcelInfo.buildings.forEach((building: IBuilding) => {
+      improvments += +building.evaluations
+        .sort((a: IEvaluation, b: IEvaluation) => compareDate(a.date, b.date))
+        .reverse()[0].value;
```

Comment:
this looks unsafe to me, what happens if there are no evaluations?

Reply:
@jdtoombs

Reply:
I'll change the check to confirm there are evaluations!